config.tests/rocksdb/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <cstddef>
+#include <rocksdb/version.h>
+
+constexpr int minimumVersion[] = {6, 6, 4};
+constexpr int version[] = {ROCKSDB_MAJOR, ROCKSDB_MINOR, ROCKSDB_PATCH};
+
+constexpr bool compareVersion(const int *version1, const int *version2, const size_t length)
+{
+    for (size_t i = 0; i < length; i++) {
+        if (version1[i] > version2[i]) {
+            return true;
+        } else if (version1[i] < version2[i]) {
+            return false;
+        }
+    }
+    return true;
+}
+
+static_assert(compareVersion(version, minimumVersion, sizeof(version) / sizeof(version[0])));
+
+int main() {}

```

Comment:
You may want to return 0 here unconditionally.  I'm not sure what `qtCompileTest()` expects but I would be surprised if it does not expect 0 return status...

Or does it just check if file compiles and doesn't care about return status?

Reply:
Yea, it only checks compilation. I initially had this as a runtime test, but that didn't work.